tests/installation/welcome.pm

```diff
@@ -98,8 +98,10 @@ sub run {
                 sles     => 's',
                 sled     => 'u',
                 sles4sap => get_var('OFW') ? 'u' : 'x',
-                hpc      => check_var('ARCH', 'x86_64') ? 'x' : 'u'
+                hpc      => check_var('ARCH', 'x86_64') ? 'x' : 'u',
```

Comment:
Optional thing, what's about finally creating something like `is_x86_64`?

Reply:
I agree. It would be great to have special subroutines for every architecture. I'll add the appropriate change.